cli/kata-overhead.go

```diff
@@ -0,0 +1,129 @@
+// +build cgo,linux
+// Copyright (c) 2014,2015,2016 Docker, Inc.
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/kata-containers/runtime/pkg/katautils"
+	"github.com/kata-containers/runtime/virtcontainers/types"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var kataOverheadCLICommand = cli.Command{
+	Name:  "kata-overhead",
+	Usage: "provides kata overhead at sandbox level",
+	ArgsUsage: `<sandbox-id> [sandbox-id...]
+
+   <sandbox-id> is your name for the instance of the sandbox.`,
+	Description: `The overhead command shows the overehad of a running Kata Sandbox`,
+	Action: func(context *cli.Context) error {
+		ctx, err := cliContextToContext(context)
+		if err != nil {
+			return err
+		}
+
+		args := context.Args()
+		if !args.Present() {
+			return fmt.Errorf("Missing container ID, should at least provide one")
+		}
+
+		for _, cID := range []string(args) {
+			if err := overhead(ctx, cID); err != nil {
+				return err
+			}
+		}
+
+		return nil
+	},
+}
+
+func overhead(ctx context.Context, containerID string) error {
+	span, _ := katautils.Trace(ctx, "overhead")
+	defer span.Finish()
+
+	kataLog = kataLog.WithField("container", containerID)
+	setExternalLoggers(ctx, kataLog)
+	span.SetTag("container", containerID)
+
+	status, sandboxID, err := getExistingContainerInfo(ctx, containerID)
+	if err != nil {
+		return err
+	}
+
+	containerID = status.ID
+
+	kataLog = kataLog.WithFields(logrus.Fields{
+		"container": containerID,
+		"sandbox":   sandboxID,
+	})
+
+	setExternalLoggers(ctx, kataLog)
+	span.SetTag("container", containerID)
+	span.SetTag("sandbox", sandboxID)
+
+	if status.State.State == types.StateStopped {
+		return fmt.Errorf("container with id %s is not running", status.ID)
+	}
+
+	initTime := time.Now().UnixNano()
+
+	initSandboxStats, initiContainersStats, err := vci.StatsSandbox(ctx, sandboxID)
```

Comment:
1) I think you mean initContainerStats?
2) initContainerStats reminds me of an init container.  Perhaps we can say initialSandboxStats and initialContainerStats instead? 

```suggestion
	initialSandboxStats, initialContainersStats, err := vci.StatsSandbox(ctx, sandboxID)
```